meta-freescale-mbl/recipes-bsp/u-boot/u-boot-fslc_%.bbappend

```diff
@@ -8,18 +8,79 @@ MBL_UBOOT_VERSION_imx6ul-pico-mbl = "v2019.10-rc1"
 
 inherit mbl-uboot-sign
 
-SRCREV = "c0c4ee5fce01ec0818c4f27ce029d9b16c8849ad"
-SRCREV_imx6ul-pico-mbl = "ee3b9f43dfdde29ba939342bbdc22a0549015979"
+SRCREV = "a1588ac8228881f9fe65539fa8e31f0ee3556864"
+SRCREV_imx6ul-pico-mbl = "dcf722ece6aad0c1512a26cdefc2f74a192fa9d2"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
-SRC_URI = "git://git.linaro.org/landing-teams/working/mbl/u-boot.git;protocol=https;nobranch=1 \
+SRC_URI = "git://git.denx.de/u-boot.git;protocol=https;nobranch=1 \
            ${@bb.utils.contains_any('PACKAGECONFIG','noconsole silent',' file://0002-set-silent-envs.patch','',d)} \
           "
 
+SRC_URI_append_imx7s-warp-mbl = " \
```

Comment:
SRC_URI_append_imx7s-warp-mbl and SRC_URI_append_imx7d-pico-mbl both contain patch files 0001-* to 0024-*. It would be preferable to define 1 symbol with these patch files and use it in both SRC_URI_append_imx7s-warp-mbl and SRC_URI_append_imx7d-pico-mbl. If we go through another round of testing then we should change this, otherwise given time constraints we can proceed with it in its current form.